tools/lsp/main.rs

```diff
@@ -176,6 +184,10 @@ struct Cli {
     /// Start the preview in full screen mode
     #[arg(long, action)]
     fullscreen: bool,
+
+    /// Show the preview UI
+    #[arg(long, default_value = "true", action = clap::ArgAction::Set)]
+    show_preview_ui: bool,
```

Comment:
Also needs a better documentation and name.
maybe `show_preview_toolbar`?

Reply:
Then we can rename it (alias). Right now we overload the term "ui" and "preview" IMO.